### ISSUE TITLE
Package Validator - URL is failing even though it opens in browser

### ISSUE THREAD

**Issue by TheCakeIsNaOH** [chocolatey/home#50]
The `projectUrl` is failing validation in `soundswitch`
https://community.chocolatey.org/packages/soundswitch/5.9.0

Here is the failing URL: `https://soundswitch.aaflalo.me/`

The validator log is showing `503 ServiceUnavailable` the response code, however, when I do an `iwr` I get a `200 ok` response code.

CC teknowledgist gep13

┆Issue is synchronized with this [GitLab issue](https://gitlab.com/chocolatey/community-infrastructure/package-validator/-/issues/269) by [Unito](https://www.unito.io)

**Comment by TheCakeIsNaOH** [comment]
This also appears to be affecting `minecraft-launcher`:
https://community.chocolatey.org/packages/minecraft-launcher/1.0.0.020210507
https://community.chocolatey.org/packages/minecraft-launcher/1.0.0.020210506

Urls affected:
https://www.minecraft.net/en-us/about-minecraft
https://account.mojang.com/documents/minecraft_eula

**Comment by gep13** [comment]
Another affected package version: https://community.chocolatey.org/packages/laps/6.2.0.20210510

Url's affected:
https://www.microsoft.com/en-us/download/details.aspx?id=46899
https://www.microsoft.com/en-us/download/details.aspx?id=46899

**Comment by TheCakeIsNaOH** [comment]
Looks like another one affected here:
https://community.chocolatey.org/packages/tailscale/1.8.0

With URL:
https://tailscale.com/

**Comment by gep13** [comment]
@TheCakeIsNaOH @flcdrg I have a fix in place for this, just waiting on it getting deployed.  Once in place, I plan to re-scan all the packages that are currently failing the package-validator, at which point, they should all hopefully go through without any issues (unless the URL's are actually invalid of course 😄 )

**Comment by gep13** [comment]
Looking through the logs on this, looks like there are other return status codes that could also be causing problems:

* MethodNotAllowed
* InternalServerError

**Comment by gep13** [comment]
@flcdrg @TheCakeIsNaOH all of the packages in this issue, expect the original one, soundswitch, is now correctly going through package-validator. I haven't been able to trace down what is going on with that package yet :-(

**Comment by gep13** [comment]
In addition to the soundswitch package, this package https://community.chocolatey.org/packages/powershelluniversal/1.5.16 is also currently failing package-validator.  I believe that these problems are linked, and if I can figure out what is going on, it should hopefully address both of them.

**Comment by gep13** [comment]
@flcdrg @TheCakeIsNaOH I have just deployed a new version of package-validator, and the soundswitch and powershelluniversal packages are now validating correctly. Looks like the way that Cloudflare responds to certain requests has changed. We were checking for Forbidden, but now it looks like we also need to check for ServiceUnavailable.

I am going to close off this issue, but if you spot any other URL's that aren't validating correctly, lets create a new issue.  Thanks again!
